sympy/stats/sampling/sample_numpy.py

```diff
@@ -0,0 +1,81 @@
+from functools import singledispatch
+
+from sympy.external import import_module
+from sympy.stats.crv_types import BetaDistribution, ChiSquaredDistribution, ExponentialDistribution, GammaDistribution, \
+    LogNormalDistribution, NormalDistribution, ParetoDistribution, UniformDistribution
+from sympy.stats.drv_types import GeometricDistribution, PoissonDistribution, ZetaDistribution
+from sympy.stats.frv_types import BinomialDistribution
+
+
+numpy = import_module('numpy')
+
+
+@singledispatch
+def do_sample_numpy(dist, size, rand_state):
+    return None
+
+
+# CRV:
+
+@do_sample_numpy.register(BetaDistribution)
```

Comment:
`do_sample_numpy` wasn't registered with `BetaDistribution` at least in my system and therefore wasn't overridden. Similar was the case with all other distributions. Anyways, `@do_sample_numpy.register(BetaDistribution)` is more stable and less risky than `@do_sample_numpy.register` and depending on type annotations.